src/main/java/com/pingcap/tikv/meta/TiDAGRequest.java

```diff
@@ -97,7 +102,7 @@ public long mask(long flags) {
 
   public void resolve() {
     getFields().forEach(expr -> expr.resolve(tableInfo));
-    getWhere().forEach(expr -> expr.resolve(tableInfo));
+    getFilter().forEach(expr -> expr.resolve(tableInfo));
```

Comment:
Why `getFilter`? What is wrong with `getWhere` ?

Reply:
Nothing too wrong, just because in `SparkSQL` environment, calling `where` as `filter` sounds more comfortable and will goes the same as `com.pingcap.tikv.predicates.ScanBuilder.ScanPlan`'s naming.